src/block/Sugarcane.php

```diff
@@ -123,4 +129,24 @@ public function place(BlockTransaction $tx, Item $item, Block $blockReplace, Blo
 
 		return false;
 	}
+
+	private function isSupportedByWater() : bool{
+		$down = $this->getSide(Facing::DOWN);
+		foreach(Facing::HORIZONTAL as $side){
```

Comment:
https://github.com/pmmp/PocketMine-MP/blob/b3b8aaddffe3b3a1eee8dcc62dd08fcd10581916/src/block/Block.php#L788

Reply:
okay let's try this